src/NServiceBus.Core/Routing/AutomaticSubscriptions/AutoSubscribe.cs

```diff
@@ -81,11 +86,20 @@ protected override Task OnStop(IMessageSession session)
 
             static async Task SubscribeToEvent(IMessageSession session, Type eventType)
             {
-                await session.Subscribe(eventType).ConfigureAwait(false);
-                Logger.DebugFormat("Auto subscribed to event {0}", eventType);
+                try
+                {
+                    await session.Subscribe(eventType).ConfigureAwait(false);
+                    Logger.DebugFormat("Auto subscribed to event {0}", eventType);
+                }
+                catch (Exception e) when (!(e is QueueNotFoundException))
+                {
+                    Logger.Error($"AutoSubscribe was unable to subscribe to event '{eventType.FullName}': {e.Message}");
+                    // swallow exception
+                }
             }
 
             List<Type> messagesHandledByThisEndpoint;
+            HashSet<Type> excludedTypes;
             static ILog Logger = LogManager.GetLogger<ApplySubscriptions>();
```

Comment:
we've changed the logger name in the v7 version. Shouldn't we do the same here?

Reply:
Isn't it a breaking change?

Reply:
logger names aren't covered by our semver policy.

Reply:
changed